export_img.py

```diff
@@ -0,0 +1,15 @@
+import sys
+import subprocess
+from pathlib import Path
+
+base = Path(".")
+
+inkscape = "C:/Program Files/Inkscape/inkscape"
```

Comment:
A co tam dát tu cestu k inkscape takto? a nemuselo by se to tam vypisovat...

```python
# kdyby měl někdo jinou cestu, může si ji sem dopsat
inkscape_paths = [
    "inkscape",
    "C:/Program Files/Inkscape/inkscape",
    "C:/Program Files (x86)/Inkscape/inkscape"
    ]

def inkscape_exists(path):
''' kontrala zda je inkscape v cestě a jde pustit '''

    try:
        subprocess.run([path, "--version"],
                       stdin=subprocess.DEVNULL,
                       stdout=subprocess.DEVNULL,
                       stderr=subprocess.DEVNULL)
        return True
    except FileNotFoundError:
        return False

def find_inkscape():
''' funkce vrátí první cestu, která funguje '''

    for path in inkscape_paths:
        if inkscape_exists(path):
            return path

inkscape = find_inkscape()
```

Reply:
Trošku složité ne? :-D Vůbec tomu nerozumím cos napsala, zítra proberen

Reply:
Jasně... :)